data/hai/hai reveal 2 start.txt

```diff
@@ -144,6 +190,29 @@ mission "Hai Rescue: Prison Role-Players"
 	# No NPCs for this one since the Hai are presumed dead.
 
 
+mission "Hai Rescue: Oonem Hint"
+	landing
+	name "A (Short) Angel of Death"
+	description "There have been some mysterious deaths on Stormhold."
+	source
+		near "Alcyone" 3
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+	destination Stormhold
+	to offer
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Ooonem: offered"
+		not "event: hai-human treaty signed"
+		random < 30
+	on offer
+		conversation
+			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower less obvious bands, you hear a pair of captains talking about the pirate world of Stormhold. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`
```

Comment:
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower, less obvious bands, you hear a pair of captains talking about the pirate world of <planet>. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`
```
OR
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower, less obvious frequencies, you hear a pair of captains talking about the pirate world of <planet>. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`
```
Is the use of `band` and not `frequency` a scientifically accurate thing? I'm just more used to hearing `frequency` used in this case, so I thought I'd point it out. Bottom has that edit; top is just grammar.

Reply:
My understanding is that in the case where frequencies close to each other are capable of being broadcast and can interfere with each other on the same system, the term band is used for each of the selection of frequencies which are spaced apart enough to all be used safely without interference.